install_config/cluster_metrics.adoc

```diff
@@ -414,20 +413,20 @@ start script based on available memory of the node on which it is scheduled.
 |The supplemental storage group to use for Cassandra.
 
 |`openshift_metrics_hawkular_ca`
-|An optional certificate file:0 used to sign the Hawkular certificate.
+|An optional certificate file: 0 used to sign the Hawkular certificate.
 
```

Comment:
Proposed description:   An optional certificate authority (CA) certificate file used to sign the Hawkular certificate.